pkg/minikube/audit/logFile.go

```diff
@@ -30,6 +31,9 @@ var currentLogFile *os.File
 // setLogFile sets the logPath and creates the log file if it doesn't exist.
 func setLogFile() error {
 	lp := localpath.AuditLog()
+	if err := os.MkdirAll(filepath.Dir(lp), 0755); err != nil {
```

Comment:
would this make a Log Dir for "minikube delete --all --purge" ? (in that case we dont wanna create dirs if users wants to purge everything)

Reply:
no, no, it will just create dir for setLogFile when it's called

"minikube delete --all --purge" was mention as to how it can be replicated, eg, `make test` w/o ~/.minikube dir structure will fail when calling setLogFile